eng/scripts/InstallVisualStudio.ps1

```diff
@@ -59,29 +65,36 @@ mkdir $intermedateDir -ErrorAction Ignore | Out-Null
 $bootstrapper = "$intermedateDir\vsinstaller.exe"
 $ProgressPreference = 'SilentlyContinue' # Workaround PowerShell/PowerShell#2138
 
-$channelUri = "https://aka.ms/vs/16/release"
-$responseFileName = "vs"
+if ("$Version" -eq "2019") {
+    $vsversion = 16;
+}
+if ("$Version" -eq "2022") {
+    $vsversion = 17;
+    $Channel = "Preview";
+}
+$channelUri = "https://aka.ms/vs/$vsversion/release"
+$responseFileName = "vs.$vsversion"
```

Comment:
vs.buildtools.json, vs.buildtools.preview.json, and vs.json files also need to be renamed

Reply:
Updated in a93a61c